src/main/java/org/folio/service/budget/BudgetService.java

```diff
@@ -139,7 +139,7 @@ private Future<Void> deleteAllocationTransactions(Budget budget, DBConn conn) {
 
     String sql ="DELETE FROM "+ getFullTableName(conn.getTenantId(), TRANSACTIONS_TABLE)
       + " WHERE  (fromFundId = '" + budget.getFundId() + "' OR toFundId = '" + budget.getFundId() + "')"
-      + " AND fiscalYearId = '" + budget.getFiscalYearId() + "' AND lower(f_unaccent(jsonb->>'transactionType'::text)) = 'allocation'";
+      + " AND fiscalYearId = '" + budget.getFiscalYearId() + "' AND jsonb->>'transactionType' = 'Allocation'";
```

Comment:
this place also can be reverted to use this new index:
https://github.com/folio-org/mod-finance-storage/blob/master/src/main/java/org/folio/service/budget/BudgetService.java#L158